ServerCore/Pages/Teams/Details.cshtml

```diff
@@ -4,58 +4,217 @@
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Team.SecondaryPhoneNumber)
+                </dd>
+            </dl>
+        </div>
+        <div class="column">
+            <div>
+                <h3>Members (@(Model.Members.Count))</h3>
+                @{ bool canAdd = Model.EventRole == ModelBases.EventRole.admin; }
+
+                @if (!Model.Event.IsTeamMembershipChangeActive)
+                {
+                    <div class="alert alert-danger" role="alert">
+                        This event is not currently open for membership changes.
+                    </div>
+                }
+                else if (Model.Members.Count >= Model.Event.MaxTeamSize)
+                {
+                    <div class="alert alert-danger" role="alert">
+                        This team is full.
+                    </div>
+                }
+                else
+                {
+                    canAdd = true;
+                }
+
+            </div>
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th>
+                            @Html.DisplayNameFor(model => model.Members[0].Member.Name)
+                        </th>
+                        <th>
+                            @Html.DisplayNameFor(model => model.Members[0].Member.Email)
+                        </th>
+                        <th>
+                            @Html.DisplayNameFor(model => model.Members[0].Member.EmployeeAlias)
+                        </th>
+                        <th></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    @foreach (var item in Model.Members)
+                    {
+                        <tr>
+                            <td>
+                                @if (Model.LoggedInUser.IsGlobalAdmin)
+                                {
+                                    <a asp-area="Identity" asp-page="/Account/Edit" asp-route-userId="@item.Member.ID" asp-route-returnUrl="@Request.GetEncodedUrl()">@Html.DisplayFor(modelItem => item.Member.Name)</a>
+                                }
+                                else
+                                {
+                                    @Html.DisplayFor(modelItem => item.Member.Name)
+                                }
+                            </td>
+                            <td>
+                                @Html.DisplayFor(modelItem => item.Member.Email)
+                            </td>
+                            <td>
+                                @if (item.Member.EmployeeAlias == null)
+                                {
+                                    <p>N/A</p>
+                                }
+                                @Html.DisplayFor(modelItem => item.Member.EmployeeAlias)
+                            </td>
+                            @if (canAdd)
```

Comment:
The ability to remove members should be tracked by a different variable.

Reply:
Done - changed to check if admin or event member changes were active